docs/reference/ml/trained-models/apis/start-trained-model-deployment.asciidoc

```diff
@@ -58,6 +58,22 @@ include::{es-ref-dir}/ml/ml-shared.asciidoc[tag=model-id]
 [[start-trained-model-deployment-query-params]]
 == {api-query-parms-title}
 
+`adaptive_allocations`::
```

Comment:
Just noticed: this isn't a query param, but a request body param. (same for update)